vscode/src/completions/get-current-doc-context.test.ts

```diff
@@ -258,39 +270,122 @@ describe('getCurrentDocContext', () => {
             resetParsersCache()
         })
 
-        it.each([
-            {
-                code: 'const restuls = {█',
-                triggerPosition: { line: 0, character: 16 },
-            },
-            {
-                code: 'const result = {\n  █',
-                triggerPosition: { line: 0, character: 15 },
-            },
-            {
-                code: 'const result = {\n    █',
-                triggerPosition: { line: 0, character: 15 },
-            },
-            {
-                code: 'const something = true\nfunction bubbleSort(█)',
-                triggerPosition: { line: 1, character: 19 },
-            },
-        ])('returns correct multiline trigger position', ({ code, triggerPosition }) => {
-            const { document, position } = documentAndPosition(code)
+        describe('with enabled dynamicMultlilineCompletions', () => {
+            it.each([
+                dedent`
+                    def greatest_common_divisor(a, b):█
+                `,
+                dedent`
+                    def greatest_common_divisor(a, b):
+                        if a == 0:█
+                `,
+                dedent`
+                    def bubbleSort(arr):
+                        n = len(arr)
+                        for i in range(n-1):
+                            █
+                `,
+            ])('detect the multiline trigger for python', code => {
+                const {
+                    tree,
+                    docContext: { multilineTrigger, multilineTriggerPosition },
+                } = prepareTest({ code, dynamicMultlilineCompletions: true, langaugeId: 'python' })
+
+                const triggerNode = tree.rootNode.descendantForPosition(asPoint(multilineTriggerPosition!))
+                expect(multilineTrigger).toBe(triggerNode.text)
+            })
 
-            const { multilineTrigger, multilineTriggerPosition } = getCurrentDocContext({
-                document,
-                position,
-                maxPrefixLength: 100,
-                maxSuffixLength: 100,
-                dynamicMultlilineCompletions: true,
+            it.each([
+                {
+                    code: 'const restuls = {█',
```

Comment:
```suggestion
                    code: 'const result = {█',
```

for consistency with the rest